projects/plugins/jetpack/modules/seo-tools/class-jetpack-seo-utils.php

```diff
@@ -41,13 +39,9 @@ public static function is_enabled_jetpack_seo( $site_id = 0 ) {
 			return false;
 		}
 
-		if ( function_exists( 'has_any_blog_stickers' ) ) {
-			// For WPCOM simple sites.
-			if ( empty( $site_id ) ) {
-				$site_id = get_current_blog_id();
-			}
-
-			return has_any_blog_stickers( array( 'business-plan', 'ecommerce-plan' ), $site_id );
+		// For WPcom sites.
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM && method_exists( 'Jetpack_Plan', 'supports' ) ) {
+			return Jetpack_Plan::supports( 'advanced-seo' );
```

Comment:
I checked the feature code and confirmed this looks like the correct one to call, and that it is supported for Pro/Business/eCommerce, which is exactly what we want.